playstore/notification.go

```diff
@@ -27,16 +27,25 @@ const (
 	OneTimeProductNotificationTypeCanceled
 )
 
+// https://developer.android.com/google/play/billing/rtdn-reference#voided-purchase
+type VoidedPurchaseProductType int
+
+const (
+	VoidedPurchaseProductTypeSubscription = iota + 1
+	VoidedPurchaseProductTypeOneTime
+)
+
 // DeveloperNotification is sent by a Pub/Sub topic.
 // Detailed description is following.
 // https://developer.android.com/google/play/billing/rtdn-reference#json_specification
 type DeveloperNotification struct {
-	Version                    string                     `json:"version"`
-	PackageName                string                     `json:"packageName"`
-	EventTimeMillis            string                     `json:"eventTimeMillis"`
-	SubscriptionNotification   SubscriptionNotification   `json:"subscriptionNotification,omitempty"`
-	OneTimeProductNotification OneTimeProductNotification `json:"oneTimeProductNotification,omitempty"`
-	TestNotification           TestNotification           `json:"testNotification,omitempty"`
+	Version                    string                      `json:"version"`
+	PackageName                string                      `json:"packageName"`
+	EventTimeMillis            string                      `json:"eventTimeMillis"`
+	SubscriptionNotification   SubscriptionNotification    `json:"subscriptionNotification,omitempty"`
+	OneTimeProductNotification OneTimeProductNotification  `json:"oneTimeProductNotification,omitempty"`
+	VoidedPurchaseNotification *VoidedPurchaseNotification `json:"voidedPurchaseNotification,omitempty"`
```

Comment:
I want it to match the other fields and not be a pointer 🙏

Reply:
OK, please check the latest commit.